Sources/Sentry/SentryTimeToDisplayTracker.m

```diff
@@ -26,7 +26,7 @@
 
 @implementation SentryTimeToDisplayTracker {
     BOOL _waitForFullDisplay;
-    BOOL _isReadyToDisplay;
+    BOOL _initialDisplayReported;
```

Comment:
I thought it's better to align the naming, because we have `initialDisplaySpan` and `fullDisplaySpan`, `_initialDisplayReported` and `_fullyDisplayedReported`, `reportInitialDisplay` and `reportFullyDisplayed`. It was confusing to me that we have `_isReadyToDisplay` and `reportReadyToDisplay` while they relate to the `initialDisplaySpan`. It's clearer to me this way, so I would keep it. The method and field is not public, so we can change it anytime.